neural_compressor/torch/algorithms/weight_only/autoround.py

```diff
@@ -12,13 +12,169 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import time
+
 import torch
 from auto_round import AutoRound  # pylint: disable=E0401
 from auto_round.calib_dataset import CALIB_DATASETS  # pylint: disable=E0401
+from auto_round.utils import get_block_names  # pylint: disable=E0401
 
+from neural_compressor.torch.algorithms import Quantizer
 from neural_compressor.torch.utils import logger
 
 
+class AutoRoundQuantizer(Quantizer):
+    def __init__(
+        self,
+        model,
+        weight_config: dict = {},
+        enable_full_range: bool = False,
+        batch_size: int = 8,
+        amp: bool = True,
+        device=None,
+        lr_scheduler=None,
+        use_quant_input: bool = True,
```

Comment:
Please note that autoround has updated the parameter name use_quant_input->enable_quanted_input.